src/mam4xx/gasaerexch.hpp

```diff
@@ -319,6 +319,91 @@ Real fuchs_sutugin(const Real &D_p, const Real &gasfreepath,
   return fuchs_sutugin;
 }
 
+KOKKOS_INLINE_FUNCTION
+void gas_aer_uptkrates_1box1gas_OD(const Real accom, const Real gasdiffus,
```

Comment:
When it is ready, we may want to remove OD for the new function name and the old function as well.

Reply:
I will delete OD. We can keep both versions of this routine for now.